src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex(center, '');
```

Comment:
The second argument in the `flex` mixin is an empty string. If the mixin expects a specific value or keyword, consider providing it or removing the argument if it's optional.